### PR TITLE
Landing page: Add menu and style features

### PR DESCRIPTION
**Pull request by jvaclavik**

### CODE REVIEW

**Comment by aalises** [comment]
Header looks a little bit weird when resizing on web :o
eeeh

File: apps/landingPage/components/Content.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../mediaQueriesConfig';
+
+export default styled.div`
+  max-width: 80vw;
+  position: relative;
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) {
```
Comment (KarinaDavtyan) [inline apps/landingPage/components/Content.js]:
```suggestion
  @media (min-width: ${defaultTokens.widthBreakpointLargeMobile) {
```

File: apps/landingPage/components/Footer.js [hunk 2]
```diff
@@ -79,20 +105,28 @@ const FooterContainer = styled.div`
   }
 `;
 
+const LinksWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  padding: 40px 0 10px 0;
+  flex-wrap: wrap;
+  @media (min-width: ${BREAKPOINTS.TABLET}px) {
```
Comment (KarinaDavtyan) [inline apps/landingPage/components/Footer.js]: Maybe removing for now if its empty

File: apps/landingPage/components/Header.js [hunk 3]
```diff
@@ -50,115 +50,108 @@ const logos = [
...
-    </Container>
+    <>
+      <Container>
+        <Content>
+          <UpperHeader>
+            <Logo src="/static/logo_white.png" alt="kiwi.com logo" />
+          </UpperHeader>
+          <Columns>
+            <Headline>
+              <Motto inverted={true} element="div">
```
Comment (KarinaDavtyan) [inline apps/landingPage/components/Header.js]: Why do we need to remove <Headline /> from orbit here?

**Comment by KarinaDavtyan** [comment]
Maybe `Content.js`, `AccentedText.js`, `Highlight.js` should go to `styledComponent/` (or any other similar name) folder? I think it would be more convenient to have separate folder for just styled components

File: apps/landingPage/components/Content.js (hunk 1 above)
Comment (jvaclavik) [reply apps/landingPage/components/Content.js]: OK

File: apps/landingPage/components/Header.js (hunk 3 above)
Comment (jvaclavik) [reply apps/landingPage/components/Header.js]: We need different font and different styling

File: apps/landingPage/components/Footer.js (hunk 2 above)
Comment (jvaclavik) [reply apps/landingPage/components/Footer.js]: Done

**Comment by jvaclavik** [comment]
> Header looks a little bit weird when resizing on web :o
> eeeh

It will be solved in another PR

**Pull request merged**